source/slang/slang-compiler.cpp

```diff
@@ -290,6 +320,32 @@ namespace Slang
         return Super::getInterface(guid);
     }
 
+    SlangResult TypeConformance::computeDependencyBasedHash(
+        slang::Checksum* outHashCode)
+    {
+        auto subtypeWitness = m_subtypeWitness->toString();
```

Comment:
It seems important for somebody to confirm that the `toString()` operation on these witnesses will return a "sufficiently unique" string.

Reply:
Deferring to a separate cleanup PR